azure/packages/azure-client/package.json

```diff
@@ -34,10 +34,10 @@
 		"build:commonjs": "fluid-build . --task commonjs",
 		"build:compile": "fluid-build . --task compile",
 		"build:docs": "fluid-build . --task api",
-		"build:esnext": "tsc --project ./tsconfig.esnext.json",
-		"build:test": "tsc --project ./src/test/tsconfig.json",
+		"build:esnext": "tsc-multi --config ../../../common/build/build-common/tsc-multi.esm.json",
```

Comment:
Is there a reason this doesn't / can't invoke fluid-build? Or is this the leaf node in the esnext build chain, where `tsc` is the leaf for `build:commonjs`?

Reply:
it's the leaf task.

Reply:
Not for this PR, but should we consider making the two builds' script chains mirror one another? I.e. either replace `tsc` with `tsc:commonjs` and `tsc:esnext` or inline `tsc` into `build:commonjs`?